tests/unit/array/storage/weaviate/test_additional_properties.py

```diff
@@ -0,0 +1,29 @@
+from docarray import Document, DocumentArray
+
+
+def test_get_additional(start_storage):
+    da = DocumentArray(
+        storage="weaviate", config={"name": "Test", "host": "localhost", "port": 8080}
+    )
+
+    with da:
+        da.extend(
+            [
+                Document(text="r0", embedding=[0, 0, 0]),
+                Document(text="r2", embedding=[2, 2, 2]),
+                Document(text="r4", embedding=[4, 4, 4]),
+                Document(text="r5", embedding=[2, 2, 2]),
+                Document(text="r6", embedding=[4, 4, 4]),
```

Comment:
```suggestion
                Document(embedding=[0, 0, 0]),
                Document(embedding=[2, 2, 2]),
                Document(embedding=[4, 4, 4]),
                Document(embedding=[2, 2, 2]),
                Document(embedding=[4, 4, 4]),
```
I don't really get what's the purpose of setting text field here